pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -270,6 +278,35 @@ func evictPods(inputPods []*v1.Pod,
 	}
 }
 
+func podToleratesTaints(pod *v1.Pod, taintsOfNodes [][]v1.Taint) bool {
+	canBeEvicted := true
+taintNodeLoop:
+	for _, taintsForNode := range taintsOfNodes {
+		if len(pod.Spec.Tolerations) < len(taintsForNode) {
+			canBeEvicted = false
+			continue
+		}
+		for _, taint := range taintsForNode {
```

Comment:
`k8s.io/kubernetes/pkg/apis/core/v1/helper` defines `TolerationsTolerateTaintsWithFilter` function which you might use instead:

```go
helper.TolerationsTolerateTaintsWithFilter(pod.Spec.Tolerations, taintsForNode, nil)
```

Reply:
Will use this!